<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_In</class>
 <widget class="QMainWindow" name="Sign_In">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	font-family:century gothic;
	font-size:24px;
	background-color:rgb(255, 217, 219)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Sign_in_pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>440</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color:rgb(165, 142, 255);
border-radius:30px;
}
#Sign_in_pushButton:hover
{
background-color:#60adff
}
</string>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Email_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Password_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Username_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Sign in as an Student with using Student word in username and Professor with Prof word"/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
